packages/odyssey-react/src/components/Heading/Heading.tsx

```diff
@@ -82,11 +82,9 @@ let Title = forwardRef<HTMLHeadingElement, Props>((props, ref) => {
   );
 });
 
-Title.displayName = "Title";
+Heading.displayName = "Heading";
 
-Title = withStyles(styles)(Title);
+Heading = withStyles(styles)(Heading);
 
-type TitleProps = ComponentProps<typeof Title>;
-export type { TitleProps as Props };
-
-export default Title;
+export type HeadingProps = ComponentProps<typeof Heading>;
+export { Heading };
```

Comment:
taking the opportunity here to rename to avoid a breaking change in the future via renaming the named export